rviz_default_plugins/src/rviz_default_plugins/displays/laser_scan/laser_scan_display.cpp

```diff
@@ -72,6 +85,7 @@ void LaserScanDisplay::processMessage(sensor_msgs::msg::LaserScan::ConstSharedPt
   if (tolerance > filter_tolerance_) {
     filter_tolerance_ = tolerance;
     tf_filter_->setTolerance(filter_tolerance_);
+    checkTolerance(filter_tolerance_.nanoseconds() * 1e-9);
```

Comment:
`filter_tolerance_` is an `rclcpp::Duration`.  Calling `nanoseconds()` on that returns an `rcl_duration_value_t`, which is an `rcutils_duration_value_t`, which is an `int64_t`.  And then we are multiplying that number by 1e-9.

I kind of feel like we should change this up a bit so that `checkTolerance` takes in an `rclcpp::Duration`, and then we do the conversion in there using `RCL_NS_TO_S`.  That seems like it will be clearer on what the intention is, and will avoid doing some math using an int64_t and a double at this level.